conclave-host/src/main/kotlin/com/r3/conclave/host/internal/attestation/EnclaveQuoteServiceGramineDCAP.kt

```diff
@@ -0,0 +1,23 @@
+package com.r3.conclave.host.internal.attestation
+
+import com.r3.conclave.common.internal.*
+import com.r3.conclave.host.internal.Native
+import com.r3.conclave.host.internal.NativeLoader
+import java.nio.ByteBuffer
+
+class EnclaveQuoteServiceGramineDCAP : EnclaveQuoteService() {
+
+    override fun getQuotingEnclaveInfo(): Cursor<SgxTargetInfo, ByteBuffer> {
+        val targetInfo = Cursor.allocate(SgxTargetInfo)
```

Comment:
I think the answer is no, because we don't need the target info for Gramine. Correct? If so then we can move the `initQuoteDCAP` into the companion object so that it's only loaded in once.

Reply:
Yes, I am still trying to make sense of this mechanism. This class is actually useless, because in the case of Gramine I could initialize the library in the related attestation class.
The actual value of targetInfo should be null, as we should not pass any target in the context of calling `getSignedQuote` in Gramine (this is not done right now). This is because Gramine retrieves the enclave info of the Quoting enclave automatically.